tests/nested_set_test.js

```diff
@@ -52,7 +52,7 @@ var tests = testCase({
 
         var oscar = new User({username: 'oscar', parentId: angela._id});
 
-        async.forEach([
+        async.forEachOfSeries([
```

Comment:
When the Users are inserted in parallel sometimes the tree gets broken

See https://travis-ci.org/velichkov/mongoose-nested-set/jobs/125070701

```
✖ rebuildTree should set lft and rgt based on parentIds
AssertionError: 17 == 9
    at Object.equal (/home/travis/build/velichkov/mongoose-nested-set/node_modules/nodeunit/lib/types.js:83:39)
    at /home/travis/build/velichkov/mongoose-nested-set/tests/nested_set_test.js:141:20
    at Array.forEach (native)
    at Query.<anonymous> (/home/travis/build/velichkov/mongoose-nested-set/tests/nested_set_test.js:112:17)
    at /home/travis/build/velichkov/mongoose-nested-set/node_modules/mongoose/node_modules/kareem/index.js:177:19
    at /home/travis/build/velichkov/mongoose-nested-set/node_modules/mongoose/node_modules/kareem/index.js:109:16
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickCallback (node.js:349:13)
```